zh/option/series/treemap.md

```diff
@@ -818,12 +800,6 @@ treemap 默认把第一个维度（Array 第一项）映射到『面积』上。
 矩形的颜色。默认从全局调色盘 [option.color](~color) 获取颜色。
 
 {{ if: ${itemStyleType} === 'normal' }}
-#${prefix} colorAlpha(number) = null
-
-<ExampleUIControlNumber step="0.01" min="0" max="1" default="1" />
-
-矩形颜色的透明度。取值范围是 0 ~ 1 之间的浮点数。
-
 #${prefix} colorSaturation(number) = null
```

Comment:
`itemStyle.colorSaturation` can also be removed for the same reason

Reply:
For this part, color saturation is a part of hsv mode. But according to https://echarts.apache.org/zh/option.html#color, all Echarts provides is rgb mode, which means if deleting `colorSaturation` developers have no ways to adjust saturation. 
However, it's not tested whether it can work properly.

Reply:
@jiawulin001 `hsl` is also supported. Check https://github.com/ecomfe/zrender/blob/master/src/tool/color.ts#L261

Reply:
Oh, that's a good one. Maybe we should also add hsl into `color` documentation?

Reply:
@jiawulin001 That will be great